app/views/layouts/application.html.erb

```diff
@@ -21,7 +21,11 @@
     <% end %>
     <main role="main">
       <div class="container">
-        <%= render 'layouts/messages' %>
+        <div id="flash">
```

Comment:
If it turns out to be necessary, doable to just move it all to the layouts/messages partial so we can avoid editing the app layout?